index.html

```diff
@@ -35,9 +35,290 @@ <h3>I am a software developer</h3>
         </section> 
        </section>
 
-       <section>
-        <div>
-            <img src="./images/Icon - Angelist.png" alt="">
+       <section class="recent">
+        <div class="title">
+            <p>My Recent works</p>
```

Comment:
- [ ] Kindly use a heading element (ie `hx` element => `<h2>`, `<h3>` ... , `<h5>`) to wrap the heading(title) of this section. It is best practice for HTML semantics.
- [ ] I noticed that the title of this section is not displaying correctly in the browser, kindly fix this, please.
### Your design
![1](https://user-images.githubusercontent.com/94551484/192064685-e01816dc-6070-48c0-a3f3-bf30c8113d15.png)
### Figma template
![f1](https://user-images.githubusercontent.com/94551484/192064738-99272f26-de45-4f0d-bf01-e2543e2d0984.png)